packages/app/components/Nav/NavBar.tsx

```diff
@@ -116,6 +116,7 @@ export default function NavBar({ courseId }: NavBarProps): ReactElement {
     courseId = profile?.courses[0].course.id;
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [defaultCourse, setDefaultCourse] = useLocalStorage(
```

Comment:
`defaultCourse` is unused -- but there's no way to destructure this without an unused variable